src/imitation/algorithms/mce_irl.py

```diff
@@ -182,7 +192,7 @@ def forward(
         self,
         observation: th.Tensor,
         deterministic: bool = False,
-    ):
+    ) -> NoReturn:
```

Comment:
Interesting . Is this type not incompatible with the superclass? How does that work?

Reply:
I think it works because `NoReturn` is the bottom type: i.e. a type that is a subtype of all other types. There's a good discussion at https://github.com/python/mypy/issues/4116